core/src/main/scala/org/dbpedia/extraction/wikiparser/NodeUtil.scala

```diff
@@ -210,4 +213,32 @@ object NodeUtil
 
         splitNodes.reverse
     }
+
+    def removeEmptyTextNode(list : List[Node]) : List[Node] = {
```

Comment:
`remove` sounds dangerous and not clear if the nodes are deleted or not :)
`copy excluding` or `copy without` sounds safer :)